platformio.ini

```diff
@@ -27,9 +27,10 @@ build_flags =
 	;-DMACHINE_FILENAME=test_drive.h ;Remove ";" from the beginning of this line and specify the machine file
 	-DCORE_DEBUG_LEVEL=0
 	-Wno-unused-variable
-	-Wno-unused-function
-        ;-DDEBUG_REPORT_HEAP_SIZE
-        ;-DDEBUG_REPORT_STACK_FREE
+	-Wno-unused-
```

Comment:
Is this a typo?

Reply:
I did not intend to change that file.  Reverted